src/plugins/unified_search/public/filter_bar/filter_editor/filter_editor.tsx

```diff
@@ -102,6 +105,10 @@ export const strings = {
     i18n.translate('unifiedSearch.filter.filterEditor.queryDslLabel', {
       defaultMessage: 'Elasticsearch Query DSL',
     }),
+  getQueryDslDocsLinkLabel: () =>
+    i18n.translate('unifiedSearch.filter.filterEditor.queryDslDocsLinkLabel', {
+      defaultMessage: 'Learn about query DSL syntax',
```

Comment:
```suggestion
      defaultMessage: 'Learn about Query DSL syntax',
```
It looks like we typically capitalize "Query DSL" in copy.